tests/src/JIT/config/benchmark/project.json

```diff
@@ -14,6 +14,10 @@
     "System.Threading": "4.0.10",
     "System.Threading.Tasks": "4.0.10",
     "System.Threading.Tasks.Parallel": "4.0.0",
+    "System.Diagnostics.Process": "4.0.0-*",
```

Comment:
There are actually lots of things in that file that use "*".    Don't know that I want to lock in a version yet.